app_feup/lib/view/Pages/general_page_view.dart

```diff
@@ -81,13 +81,16 @@ abstract class GeneralPageViewState extends State<StatefulWidget> {
       bottom: PreferredSize(
         preferredSize: Size.zero,
         child: Container(
-          margin: EdgeInsets.only(left: borderMargin, right: borderMargin),
           color: Theme.of(context).dividerColor,
+          margin: EdgeInsets.only(left: borderMargin, right: borderMargin),
           height: 1.5,
         ),
       ),
       elevation: 0,
-      iconTheme: IconThemeData(color: Theme.of(context).accentColor),
+      iconTheme: IconThemeData(
+          color: Theme.of(context).brightness == Brightness.light
+              ? Theme.of(context).primaryColor
+              : Colors.white),
```

Comment:
If you accept the dividing the icons colors, just change this to this
```suggestion
      iconTheme: Theme.of(context).accentIconTheme,
```